src/Umbraco.Web.UI.NetCore/Umbraco.Web.UI.NetCore.csproj

```diff
@@ -61,6 +62,23 @@
           <SubType>Designer</SubType>
         </Content>
         <Content Remove="wwwroot\Web.config" />
+        <Content Remove="Umbraco\PartialViewMacros\Templates\Gallery.cshtml" />
```

Comment:
Do we need all these remove statements?

Reply:
Nope.. I will clean up in a later PR regarding the publish for alpha